donkeycar/utilities/TrackSpeedPlanner/deploy-pi-tornado.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Tornado-based Raspberry Pi deployment script for Path Data Visualizer & Editor
+# Uses Python instead of Node.js for better Pi compatibility
+
+set -e
+
+echo "🐍 Setting up Path Data Visualizer on Raspberry Pi (Python/Tornado)..."
+
+# Clean up any previous installation
+if [ -f "uninstall-pi.sh" ]; then
+    echo "🧹 Cleaning up previous installation..."
+    chmod +x uninstall-pi.sh
+    ./uninstall-pi.sh
+    echo ""
+fi
+
+# Check Python installation
+if ! command -v python3 &> /dev/null; then
+    echo "❌ Python 3 not found. Installing..."
+    sudo apt-get update
+    sudo apt-get install -y python3 python3-pip
+fi
+
+echo "✅ Python version: $(python3 --version)"
+
+# Install Tornado if not available
+echo "📦 Installing Python dependencies..."
+if ! python3 -c "import tornado" 2>/dev/null; then
+    python3 -m pip install --user tornado
+    echo "✅ Tornado installed"
+else
+    echo "✅ Tornado already available"
+fi
+
+# Make scripts executable
+chmod +x server-tornado.py
+
+# Create startup script
+echo "⚙️ Creating startup script..."
```

Comment:
Can we rather check in this python script instead of generating it here from the shell script?

Reply:
We've now removed the entire install script so this is redundant